docs/sphinx/developers/components.txt

```diff
@@ -255,11 +184,82 @@ tests) are included in this component.  These tests are primarily run by the
+now available as separate build dependencies:
+
+- :ref:`OME Common <ome-common>`
+- :ref:`OME MDB Tools (Java) <forks-mdbtools>`
+- :ref:`OME Apache Jakarta POI <forks-poi>`
+
+Decoupled OME data model components:
+
+- :ref:`OME-XML <ome-xml>`
+- :ref:`Specification <specification>`
+- :ref:`XSD-FU <xsd-fu>`
+
+.. _ome-common:
+
+OME Common (`Java <https://github.com/ome/ome-common-java>`_ /
+`C++ <https://github.com/ome/ome-files-cpp>`_):
+
+Provides I/O classes that unify reading from files on disk, streams or files
+in memory, compressed streams, and non-file URLs.  The primary entry points
+are :common_javadoc:`Location <loci/common/Location.html>`,
+:common_javadoc:`RandomAccessInputStream <loci/common/RandomAccessInputStream.html>`
+(for reading), and :common_javadoc:`RandomAccessOutputStream
+<loci/common/RandomAccessOutputStream.html>` (for writing).
+
+In addition to I/O, there are several classes to assist in working with XML
+(:common_javadoc:`XMLTools <loci/common/xml/XMLTools.html>`), date/timestamps
+(:common_javadoc:`DateTools <loci/common/DateTools.html>`), logging configuration
+(:common_javadoc:`DebugTools <loci/common/DebugTools.html>`), and byte arithmetic
+(:common_javadoc:`DataTools <loci/common/DataTools.html>`).
+
+.. _forks-mdbtools:
+
+`OME MDB Tools (Java port) <https://github.com/ome/ome-mdbtools>`_:
+
+This is a fork of the `mdbtools-java
+<http://mdbtools.cvs.sourceforge.net/viewvc/mdbtools/mdbtools-java>`_ project.
+There are numerous bug fixes, as well as changes to reduce the memory required
+for large files.  There are no dependencies on other components.
+
+.. _forks-poi:
+
+`OME Apache Jakarta POI <https://github.com/ome/ome-poi>`_:
 
-:source:`xsd-fu (XSD-FU) <components/xsd-fu>`:
+This is a fork of `Apache POI <http://poi.apache.org>`_, which allows reading
+of Microsoft OLE document files.  We have made substantial changes to support
+files larger than 2GB and reduce the amount of memory required to open a file.
+I/O is also handled by classes from :ref:`ome-common <ome-common>`, which
+allows OLE files to be read from memory.
+
+.. _ome-xml:
+
+`OME-XML Java library <https://github.com/ome/ome-model/tree/master/ome-xml>`_:
+
+This component contains classes that represent the OME-XML schema.  Some
+classes are committed to the Git repository, but the majority are generated at
+build time by using :ref:`xsd-fu <xsd-fu>` to parse the
+:ref:`OME-XML schema files <specification>`.  Classes from this component are
+used by Bio-Formats to read and write OME-XML, but they can also be used
+independently.
+
+.. _specification:
+
+`Model specification <https://github.com/ome/ome-model/tree/master/specification>`_:
+
+All released and in-progress OME-XML schema files are contained in this
+component.  The specification component is also the location of all XSLT
+stylesheets for converting between OME-XML schema versions, as well as example
+OME-XML files in each of the released schema versions.
+
+.. _xsd-fu:
```

Comment:
Briefly, following https://github.com/ome/ome-model/pull/2, `xsd-fu` is no longer a proper component, is no longer exposed and consumed and eventually might migrate inside `ome-xml` as a source generation tool. I would be inclined to remove from this page and the related `developers/xsd-fu` one. The latter can be resurrected as part of the model and formats documentation when it gets migrated to the ome-model repo.